core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/CommonLoadBalancer.scala

```diff
@@ -248,16 +274,30 @@ abstract class CommonLoadBalancer(config: WhiskConfig,
           // notice here that the activationPromises is not touched, because the expectation is that
           // the active ack is received as expected, and processing that message removed the promise
           // from the corresponding map
+          logging.info(this, s"received completion ack for '$aid', system error=${isSystemError}")(tid)
```

Comment:
This info log would be logged for happy path i.e. for all successful ack activations? If yes should we make it debug or it helps in debugging some case?

Reply:
The regular completion ack processing already created an info log before. Unless you have evidence that it should be removed, I would like to keep it because it already helped me in production systems to debug problems.

Reply:
Missed that aspect. Then we can keep it as is